packages/vats/src/vat-bank.js

```diff
@@ -226,6 +226,7 @@ export function buildRootObject(_vatPowers) {
               const amt = AmountMath.make(brand, BigInt(value));
               updater.updateState(amt);
             };
+            Far('balanceUpdater', balanceUpdater);
```

Comment:
Ooh, single-function remotables? Should we follow the pattern of
```js
const balanceUpdater = Far('balanceUpdater', ...)
```
for this, to match the way we build object-based remotables?

Reply:
Yes, that's better. Done.